src/components/PolarisTestProvider/tests/PolarisTestProvider.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {mount} from '@shopify/react-testing';
-import {PolarisTestProvider} from '../PolarisTestProvider';
+import {PolarisTestProvider} from '../..';
```

Comment:
This  doesn't seem right. Don't import from indexes

`import PolarisTestProvider from '../PolarisTestProvider';`